test/test_helper.rb

```diff
@@ -89,9 +89,20 @@ def after_teardown
   end
 end
 
+module TestCaseNetworkStubExtensions
+  def setup
+    Resolv::DNS.any_instance.stubs(:getname).returns("stubbed-host.example.com")
+    Resolv::DNS.any_instance.stubs(:getnames).returns(["stubbed-host.example.com"])
+    Resolv::DNS.any_instance.stubs(:getaddress).returns("127.0.0.13")
```

Comment:
As said on IRC, I think we should raise an exception and don't pretend it exists:
```
[4] pry(main)> Resolv.getaddress('doesnotexist.example.com')
Resolv::ResolvError: no address for doesnotexist.example.com
from /usr/share/ruby/resolv.rb:94:in `getaddress'
```